### DISCUSSION TITLE
'RankTwoAux' is not a registered object

### DISCUSSION THREAD

**Discussion by pshen20127** [idaholab/moose#25246] [Q&A Modules: Solid mechanics] [answered]
Hello

I submitted a job but there was an error like below. Do you know what's wrong with it?
![Screenshot from 2023-08-18 11-52-33](https://github.com/idaholab/moose/assets/81499506/4129f95b-c3a9-469f-84bf-1897ed6dad20)

Thanks
Pengfei

**Comment by GiudGiud** [comment]
Hello

The RankTwoAux is in the tensor mechanics module. You need to include this module in the Makefile of the app you are using and re-compile before you can use it

Please consult the guidelines before posting:
- please copy paste text instead of screenshots.
- please search other threads before creating your own. This was asked last week

**Reply by pshen20127** [reply]
I did include tensor mechanics module. But it still has that error.
![Screenshot from 2023-08-19 15-37-58](https://github.com/idaholab/moose/assets/81499506/c44a0bb3-9573-46b2-a363-6ce2e98037bd)

**Reply by GiudGiud** [reply, marked as answer]
make clobberall
then
make -j6
again

**Reply by pshen20127** [reply]
Thank you. It's fixed.
